README.md

```diff
@@ -58,9 +58,7 @@ poetry install
 ```bash
 # Create .env file for your API keys
 cp .env.example .env
-
-export OPENAI_API_KEY='your-api-key-here' # Get a key from https://platform.openai.com/
```

Comment:
What's the motivation for deleting these?

Reply:
> What's the motivation for deleting these?

This is fixed in [1f019c5](https://github.com/virattt/ai-hedge-fund/pull/18/commits/1f019c5c9770478d81a78cb4d8b46203be37437c).

Reply:
I just realized that export is only in the current terminal. Maybe we should say to edit the `.env` file since that is used instead of the export.

Reply:
This is reverted in [1f019c5](https://github.com/virattt/ai-hedge-fund/pull/18/commits/1f019c5c9770478d81a78cb4d8b46203be37437c). However, I realized that export is only in the current terminal. Since we are using the environment variables file, maybe we should say to edit the .env file or use the exports:

```
3. Set up your environment variables:
```bash
# Create .env file for your API keys
cp .env.example .env
# Update the .env file to include your API keys or export the variables in your current terminal:

export OPENAI_API_KEY='your-api-key-here' # Get a key from https://platform.openai.com/
export FINANCIAL_DATASETS_API_KEY='your-api-key-here' # Get a key from https://financialdatasets.ai/
```

WDYT?

Reply:
I have already proactively implemented this suggestion in fe6847df0de96ce9ef7210319b0a7cf029298924.